MaxText/layers/models.py

```diff
@@ -151,6 +152,9 @@ def get_decoder_layer(self):
     elif self.config.model_name[0:6] == "llama2":
       from layers import llama2
       return llama2.LlamaDecoderLayer
+    elif self.config.model_name[0:5] == "gamma":
```

Comment:
Can you look for gamma- (and above for llama2-). Just imaginging when we version up gamma.